src/app/grid-cellEditing/grid-cellEditing.component.html

```diff
@@ -25,11 +25,6 @@ <h3>Grid with primary key ProductID</h3>
     <igx-column field="ReorderLevel" width="200px" [sortable]="true" [filterable]="true" [editable]="true" [dataType]="'number'" [hasSummary]="false">
     </igx-column>
     <igx-column field="ProductName" width="200px" [groupable]="kk" [header]="pname" [sortable]="true" [dataType]="'string'" [editable]="true" [resizable]="true">
-        <ng-template igxCellValidationError let-cell='cell'>
```

Comment:
Isn't this needed in the sample?

Reply:
Nope, this is checking for the custom `forbiddenName` error, which is only set up in the validation sample. Also was using `formGroup` prop which doesn't exist -> type error.